signup/tool/src/java/org/sakaiproject/signup/tool/jsf/SignupMeetingsBean.java

```diff
@@ -591,8 +591,14 @@ private void markingRecurMeetings(List<SignupMeetingWrapper> smList) {
 	 *         current user.
 	 */
 	public boolean isAllowedToDelete() {
-		if (getSignupMeetings() == null)
+		if (sakaiFacade.isAllowedSite(sakaiFacade.getCurrentUserId(), SakaiFacade.SIGNUP_DELETE_SITE, sakaiFacade.getCurrentLocationId())) {

```

Comment:
Yes and Yes. It's a frustrating bit of permissions code.

I have a separate JIRA open for the "other sites" logic .... Other sites integration is a feature of the tool that I believe we should begin removing as it is clearly non-performant when your instructors are members of hundreds of sites.